api/src/test/java/org/jmisb/api/klv/KlvParserTest.java

```diff
@@ -62,6 +64,54 @@ private void doParse() throws KlvParseException {
         assertTrue(rawMessage.getIdentifiers().isEmpty());
     }
 
+    @Test
+    public void checkParseStream() throws KlvParseException {
+        doParseStream();
+    }
+
+    private void doParseStream() throws KlvParseException {
```

Comment:
It would also be great to write a test demonstrating error handling works as expected (e.g., a KlvParseException gets added to the output list).

Reply:
So, looking into this... since the broken out libraries (st0601, st0903), there isn't anything that just inside `api` that would throw a KlvParseException... everything gets parsed into an Unknown RawMisbMessage.java.

Reply:
I can think of two ways to deal with this - a lot of mocking, or a message factory that just throws. I've added the later as a proposed inclusion into this PR at https://github.com/ruckc/jmisb-streaming-klv/pull/1